package.json

```
@@ -18,9 +18,6 @@
     "start": "es-dev-server --node-resolve --dedupe --watch --open",
     "test": "npm run lint && npm run test:headless && npm run test:axe && npm run test:diff",
```

```suggestion
    "test": "npm run lint && npm run test:headless && npm run test:axe",
```

The README also says this which can be changed:
> # lint, unit test and visual-diff test